source/partials/release_notes/_release-17-9-0.md.erb

```diff
@@ -0,0 +1,78 @@
+<h4>Improvements</h4>
+* <%= link_to_commit '928c5856c2bc73aa92c1a8eb1a86d8094a9fe806', 'Grouping elastic profile by plugin-id', 3772%>
+
+  We have created an organized view of the elastic agent profiles. Now, you can view them by the type of elastic agent plugin (Docker Swarm, OpenStack, ECS etc.), making it easier to find a particular profile.
+
+  <figure class='shrink-figure-70'>
+      <%= image_tag 'releases/17.9.0/elastic-agent-grouping-by-plugin-id.png', alt: 'Group elastic profile by plugin-id' %>
+  </figure>
+
+* <%= link_to_commit 'ca7c3fa232087b67553e7d30bbfb409ba5aab49d', 'Addition of a request processor to allow plugin authors to query information from the GoCD server', 3401%>
```

Comment:
Add a link to `https://plugin-api.gocd.org/current/authorization/#get-server-info` ?